### PR TITLE
Add basic search for campus and courses

### PR DESCRIPTION
**Pull request by lucernae**
Add search capabilities for campus and courses.
Don't forget to rebuild index

### CODE REVIEW

File: deployment/docker-prod/REQUIREMENTS.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Django>=1.7,<1.8
+django
```
Comment (cchristelis) [inline deployment/docker-prod/REQUIREMENTS.txt]: why this change?

Comment (lucernae) [inline deployment/docker-prod/REQUIREMENTS.txt]: In my local dev env, migration raises errors because of this (when I execute it through manage.py)

Unhandled exception in thread started by <function wrapper at 0x7f5e59a48c80>
Traceback (most recent call last):
  File "/home/lucernae/WorkingDir/Projects/feti/venv/local/lib/python2.7/site-packages/django/utils/autoreload.py", line 222, in wrapper
    fn(*args, **kwargs)
  File "/home/lucernae/WorkingDir/Projects/feti/venv/local/lib/python2.7/site-packages/django/core/management/commands/runserver.py", line 107, in inner_run
    self.check_migrations()
  File "/home/lucernae/WorkingDir/Projects/feti/venv/local/lib/python2.7/site-packages/django/core/management/commands/runserver.py", line 159, in check_migrations
    executor = MigrationExecutor(connections[DEFAULT_DB_ALIAS])
  File "/home/lucernae/WorkingDir/Projects/feti/venv/local/lib/python2.7/site-packages/django/db/migrations/executor.py", line 17, in __init__
    self.loader = MigrationLoader(self.connection)
  File "/home/lucernae/WorkingDir/Projects/feti/venv/local/lib/python2.7/site-packages/django/db/migrations/loader.py", line 48, in __init__
    self.build_graph()
  File "/home/lucernae/WorkingDir/Projects/feti/venv/local/lib/python2.7/site-packages/django/db/migrations/loader.py", line 173, in build_graph
    self.load_disk()
  File "/home/lucernae/WorkingDir/Projects/feti/venv/local/lib/python2.7/site-packages/django/db/migrations/loader.py", line 103, in load_disk
    migration_module = import_module("%s.%s" % (module_name, migration_name))
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/lucernae/WorkingDir/Projects/feti/django_project/feti/migrations/0002_auto_20150424_1300.py", line 7, in <module>
    class Migration(migrations.Migration):
  File "/home/lucernae/WorkingDir/Projects/feti/django_project/feti/migrations/0002_auto_20150424_1300.py", line 18, in Migration
    preserve_default=True,
TypeError: __init__() got an unexpected keyword argument 'preserve_default'

and when I upgrade django it solves the problem.
So, what do you think I should do?
